pkg/rollup/resolution.go

```diff
@@ -0,0 +1,182 @@
+// This file and its contents are licensed under the Apache License 2.0.
+// Please see the included NOTICE for copyright information and
+// LICENSE for a copy of the license.
+
+package rollup
+
+import (
+	"context"
+	"fmt"
+	"sort"
+	"sync"
+	"time"
+
+	"github.com/timescale/promscale/pkg/log"
+	"github.com/timescale/promscale/pkg/pgxconn"
+)
+
+const (
+	originalSchema          = "prom_data"
+	upperLimit              = 5000 // Maximum samples allowed
+	assumedScrapeInterval   = time.Second * 30
+	refreshRollupResolution = time.Minute * 30
```

Comment:
Got confused with this name and what it does. Maybe  just `refreshRollupInterval`? 
Btw naming of the file is `resolution` and the logic in it has mostly to deal with finding the right rollups to use.
I think this points that we need to agree on naming here. It does seem that we use `rollup` term interchangeably with `resolution` and I think it brings confusion.

Reply:
Since this file mostly helps us decide which rollups to be used for the given query ranges, I am renaming the file and the struct to `Decider{}`.